src/main/java/com/solvd/pages/ios/LoginPageIOS.java

```diff
@@ -15,20 +16,25 @@ public class LoginPageIOS extends LoginPage{
     ExtendedWebElement passwordForm;
     @ExtendedFindBy(accessibilityId = "test-LOGIN")
     ExtendedWebElement loginButton;
+    @ExtendedFindBy(iosPredicate = "name == \"test-Error message\"")
+    ExtendedWebElement errorMessage;
```

Comment:
all properties of class should be `private`. we shouldn't give possibility to use element directly in the tests
```suggestion
    @ExtendedFindBy(accessibilityId = "test-LOGIN")
    ExtendedWebElement loginButton;
    
    @ExtendedFindBy(iosPredicate = "name == \"test-Error message\"")
    ExtendedWebElement errorMessage;
```

Reply:
How could I miss that one...Thanks!